wine.py

```diff
@@ -0,0 +1,56 @@
+def score_wine_Collector_method(hand, standard_tokens_for_trail):
+  """Uses the Collector rule to calculate the number of points for wine tokens
+  
+  Each wine is worth 1 point for each different type of food you have at least one of
+
+  Could be called any time but to reflect the real game there is no running 
+  total of points; just a single calculation and comparison after the final ant 
+  has reached the anthill.
```

Comment:
This sentence is correct, but not relevant to this function. This function doesn't really care when it is being called.